Example/TABTestKit_ExampleUITests/AppContext.swift

```diff
@@ -0,0 +1,22 @@
+//
+//  AppContext.swift
+//  TABTestKit_ExampleUITests
+//
+//  Created by Suyash Srijan on 05/11/2018.
+//  Copyright © 2018 CocoaPods. All rights reserved.
+//
+
+import XCTest
+import TABTestKit
+
+protocol AppContext {}
+extension AppContext where Self: TestBase {
```

Comment:
Don't think we need to constraint this to `Self: TestBase` because we're not using anything from `TestBase` in this extension?